doc/classes/AnimationNodeAdd2.xml

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AnimationNodeAdd2" inherits="AnimationNode" category="Core" version="3.2">
 	<brief_description>
+		Blends two animations additively inside of an [AnimationNodeBlendTree].
 	</brief_description>
 	<description>
+		A resource to add to an [AnimationNodeBlendTree]. Blends two animations additively based on an amount value in the [0.0, 1.0] range.
```

Comment:
The range should likely be given with `[code][0.0, 1.0][/code]`, otherwise I think the `makerst.py` parser will choke on these square brackets.